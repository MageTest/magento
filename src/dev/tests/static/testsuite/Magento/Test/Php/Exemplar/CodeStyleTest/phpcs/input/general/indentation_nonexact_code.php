<?php
$spaces = 1;
if (true) {
    $spaces = 2;
     $spaces = 3;
}
