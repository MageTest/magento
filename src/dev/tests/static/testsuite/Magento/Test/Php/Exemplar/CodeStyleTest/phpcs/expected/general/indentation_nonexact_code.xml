<?xml version="1.0" encoding="UTF-8"?>
<config>
    <incomplete/>
    <error line="5"/>
</config>
